addon/adapters/github.js

```diff
@@ -19,7 +19,8 @@ export default RESTAdapter.extend({
   }),
 
   pathForType(type) {
-    return Ember.String.camelize(Ember.String.pluralize(type.replace('github', '')));
+    return Ember.String.camelize(Ember.String.pluralize(type.replace('github', '')))
+      .replace('organizations', 'orgs');
```

Comment:
Why would you do this replace here rather than in the `github-organizations` adapter? Are there other adapters that use `organizations`?